### PR TITLE
Adding Netatmo warnings for OH4

### PR DESCRIPTION
**Pull request by clinique**

### CODE REVIEW

File: distributions/openhab/src/main/resources/bin/update.lst [hunk 1]
```diff
@@ -102,6 +102,8 @@ ALERT;JavaScript: JavaScript engines have changed their MIME types: NashornJS is
 ALERT;JavaScript NashornJS: NashornJS has been removed from core and isn't included by default. If you still need or want to use it, you can install it as an addon.
 ALERT;JavaScript Scripting Automation: The old "metadata" and "itemchannellink" APIs have been replaced by a new API with extended functionality on the "items" namespace.
 ALERT;JavaScript Scripting Automation: ItemHistory: historicState, maximumBetween, maximumSince, minimumBetween, minimumSince & previousState now return a HistoricItem instead of just the state. previousStateTimestamp has been removed, the timestamp is available on the HistoricItem.
+ALERT:Netatmo: due to API authorization process change scheduled on the 04/17/2023 - refresh Token is no more stored in configuration, thus can be removed from things config files. If it remains, it'll be ignored.
```
Comment (lolodomo) [inline distributions/openhab/src/main/resources/bin/update.lst:105]: due to API authorization process change scheduled on the **2023/04/17,** refresh Token is no more stored in **thing** configuration, thus can be removed from things config files. If it remains, it'll be ignored.

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: distributions/openhab/src/main/resources/bin/update.lst (hunk 1 above)
Comment (clinique) [reply distributions/openhab/src/main/resources/bin/update.lst]: Done

File: distributions/openhab/src/main/resources/bin/update.lst [hunk 2]
```diff
@@ -102,6 +102,8 @@ ALERT;JavaScript: JavaScript engines have changed their MIME types: NashornJS is
 ALERT;JavaScript NashornJS: NashornJS has been removed from core and isn't included by default. If you still need or want to use it, you can install it as an addon.
 ALERT;JavaScript Scripting Automation: The old "metadata" and "itemchannellink" APIs have been replaced by a new API with extended functionality on the "items" namespace.
 ALERT;JavaScript Scripting Automation: ItemHistory: historicState, maximumBetween, maximumSince, minimumBetween, minimumSince & previousState now return a HistoricItem instead of just the state. previousStateTimestamp has been removed, the timestamp is available on the HistoricItem.
+ALERT:Netatmo: due to API authorization process change scheduled on the 2023/04/17 - refresh Token is no more stored in thing configuration, thus can be removed from things config files. If it remains, it'll be ignored.
```
Comment (wborn) [inline distributions/openhab/src/main/resources/bin/update.lst:105]:
```suggestion
ALERT;Netatmo: Due to API authorization process change scheduled on the 2023/04/17 - refresh Token is no more stored in thing configuration, thus can be removed from things config files. If it remains, it'll be ignored.
```

File: distributions/openhab/src/main/resources/bin/update.lst [hunk 3]
```diff
@@ -102,6 +102,8 @@ ALERT;JavaScript: JavaScript engines have changed their MIME types: NashornJS is
 ALERT;JavaScript NashornJS: NashornJS has been removed from core and isn't included by default. If you still need or want to use it, you can install it as an addon.
 ALERT;JavaScript Scripting Automation: The old "metadata" and "itemchannellink" APIs have been replaced by a new API with extended functionality on the "items" namespace.
 ALERT;JavaScript Scripting Automation: ItemHistory: historicState, maximumBetween, maximumSince, minimumBetween, minimumSince & previousState now return a HistoricItem instead of just the state. previousStateTimestamp has been removed, the timestamp is available on the HistoricItem.
+ALERT;Netatmo: Due to API authorization process change scheduled on the 2023/04/17 - refresh Token is no more stored in thing configuration, thus can be removed from things config files. If it remains, it'll be ignored.
+ALERT;Netatmo: New scope has been added for the introduction of the Carbon Monoxide Alarm. Authorization process has to be replayed (and former refreshToken can be removed from things config files -- see above).
```
Comment (wborn) [inline distributions/openhab/src/main/resources/bin/update.lst:106]:
```suggestion
ALERT;Netatmo Binding: Due to API authorization process change scheduled on the 2023/04/17 - refresh Token is no more stored in thing configuration, thus can be removed from things config files. If it remains, it'll be ignored.
ALERT;Netatmo Binding: New scope has been added for the introduction of the Carbon Monoxide Alarm. Authorization process has to be replayed (and former refreshToken can be removed from things config files -- see above).
```

**Review by wborn — APPROVED** [review]
Thanks!

**Pull request merged**